include/constants.h

```diff
@@ -107,6 +109,8 @@ namespace reinforcement_learning {  namespace value {
       const char *const LEARNING_MODE_LOGGINGONLY = "LOGGINGONLY";
       const char *const CONTENT_ENCODING_IDENTITY = "IDENTITY";
       const char *const CONTENT_ENCODING_DEDUP = "DEDUP";
+      const char* const HEADER_NAME = "HEADER_NAME";
```

Comment:
Please elaborate on `HEADER_NAME` and `DEFAULT_HEADER`. This is a specific header with a specific purpose. What is the header for? How would you name this if there were two different header values?

Reply:
renamed